cumulus-model/src/main/java/mil/army/usace/hec/cumulus/client/model/CumulusObjectMapper.java

```diff
@@ -0,0 +1,45 @@
+package mil.army.usace.hec.cumulus.client.model;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.json.JsonReadFeature;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import java.io.IOException;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+
+public final class CumulusObjectMapper {
+
+    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper()
+        .registerModule(new JavaTimeModule()
+            .addDeserializer(ZonedDateTime.class, new CumulusDateTimeDeSerializer()))
+        .configure(JsonReadFeature.ALLOW_UNESCAPED_CONTROL_CHARS.mappedFeature(), true)
+        .configure(JsonReadFeature.ALLOW_MISSING_VALUES.mappedFeature(), true);
+
+    private CumulusObjectMapper() {
+        throw new AssertionError("Utility class");
+    }
+
+    public static <T> T mapJsonToObject(String json, Class<T> classObject) throws IOException {
+        return OBJECT_MAPPER.readValue(json, classObject);
+    }
+
+    public static <T> List<T> mapJsonToListOfObjects(String json, Class<T> classObject) throws IOException {
+        return OBJECT_MAPPER.readValue(json, OBJECT_MAPPER.getTypeFactory().constructCollectionType(List.class, classObject));
+    }
+
+    private static class CumulusDateTimeDeSerializer extends JsonDeserializer<ZonedDateTime> {
+
+        private final DateTimeFormatter fmt = DateTimeFormatter.ISO_ZONED_DATE_TIME;
```

Comment:
Can you show me cumulus serialized date/times in iso format?

Reply:
I based that off the example returned json seen here: https://petstore.swagger.io/?url=https://raw.githubusercontent.com/USACE/cumulus-api/stable/docs/apidoc.yaml#/Acquirables/list-acquirable-files

Reply:
Oh nice,     "datetime": "2022-03-23T18:58:32.974Z", full specification!